modules/openapi-generator/src/main/resources/kotlin-client/data_class_req_var.mustache

```diff
@@ -1,4 +1,4 @@
 {{#description}}
     /* {{{description}}} */
 {{/description}}
-    val {{{name}}}: {{#isEnum}}{{classname}}.{{nameInCamelCase}}{{/isEnum}}{{^isEnum}}{{{dataType}}}{{/isEnum}}
\ No newline at end of file
+    @Json(name = "{{{baseName}}}") val {{{name}}}: {{#isEnum}}{{classname}}.{{nameInCamelCase}}{{/isEnum}}{{^isEnum}}{{{dataType}}}{{/isEnum}}
```

Comment:
Is it possible to remove the newline at the end of the file? Looks like this is adding an extra line between properties in the generated output, and this formatting may not be desirable to consumers.

I wonder if putting the new annotation on it's own line would make it look as though the additional newline is expected.

Reply:
I did both: Remove the new at the end-of-file (having the comma on its own line was really ugly) and putting the annotation on its own line (not sure about that, let me know what you think).